pilot/pkg/xds/debug.go

```diff
@@ -158,10 +158,10 @@ func (s *DiscoveryServer) Syncz(w http.ResponseWriter, _ *http.Request) {
 	syncz := make([]SyncStatus, 0)
 	s.adsClientsMutex.RLock()
 	for _, con := range s.adsClients {
-		con.mu.RLock()
-		if con.node != nil {
+		node := con.node
+		if node != nil {
 			syncz = append(syncz, SyncStatus{
-				ProxyID:       con.node.ID,
+				ProxyID:       node.ID,
 				IstioVersion:  con.node.Metadata.IstioVersion,
```

Comment:
Should this be 

```
				IstioVersion:  node.Metadata.IstioVersion,
```

as also got updated in the  https://github.com/istio/istio/pull/26260?

Reply:
changed, thanks for catching.